docs/rfcs/active/dashboards/1778-dashboards.md

```diff
@@ -0,0 +1,117 @@
+- Start Date: 2020-08-03
```

Comment:
Quick question but do you think we should name the directory `1778-dashboards` too? Or do you think we should have these directory groupings? I'm fine either way, but it may make something harder to find if they're one level down.

(I'm not saying get rid of the directory, btw, I think this is good because you have images)

Reply:
Thanks for the suggestion, I think it would be better to have the directory name as `1778-dashboards`. And I guess, I can get rid of the `/imgs` subdirectory in that case. I will convert it to below structure:

```
1778-dashboards
|
 -- README.md
 -- dashboards_graph.png
 -- high_level_design.png
```

Reply:
Can you please update the RFC template/guide accordingly?

Reply:
Will do